napari/plugins/_tests/test_hook_callers.py

```diff
@@ -0,0 +1,468 @@
+import pytest
+from pluggy import HookCallError, HookspecMarker, PluginValidationError
+from pluggy.hooks import HookImpl
+
+from napari.plugins import _hook_callers, PluginManager
+from napari_plugins import HookimplMarker
+
+hookspec = HookspecMarker("example")
+hookimpl = HookimplMarker("example")
+
+
```

Comment:
these tests are mostly from pluggy, but import from our local overrides (`_hook_callers` and `PluginManager`) instead.

Reply:
We can leave these as-is for now, but if so, I'd like an issue created to naparify these tests at some point. I think the pluggy devs loved short names far too much.